pipstrap.py

```
@@ -124,6 +125,17 @@ def read_chunks(response, chunk_size):
 
 
 def main():
+    pip_location = check_output('command -v pip', shell=True).strip()
+    pip_version = check_output('pip --version', shell=True).split()[1]
+    pip_version = StrictVersion(pip_version)
```

I usually just compare tuples, but the code behind this looks simple and might save us a mistake if a beta version ever ends up getting into the mix. The only thing that makes me nervous is that the whole `distutils.version` module is entirely undocumented in the stdlib docs. But it's old enough that I doubt it will change, and if things move or change much, it should fail loudly.